src/System.CommandLine.Tests/System.CommandLine.Tests.csproj

```
@@ -1,6 +1,6 @@
-<Project Sdk="Microsoft.NET.Sdk">
+﻿<Project Sdk="Microsoft.NET.Sdk">
   <PropertyGroup>
-    <TargetFramework>netcoreapp2.1</TargetFramework>
+    <TargetFrameworks>netcoreapp2.1;net472</TargetFrameworks>
```

@wojtpl2 you can use an MSBuild property to make net472 target Windows-only

```suggestion
    <TargetFrameworks>netcoreapp2.1</TargetFrameworks>
    <TargetFrameworks Condition="'$(OS)' == 'Windows_NT'">$(TargetFrameworks);net472</TargetFrameworks>
```